cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/AbstractSimulator.java

```diff
@@ -13,24 +12,24 @@
  * Created by okutane on 16/08/16.
  */
 class AbstractSimulator {
-    private Set<Node> suspiciousPoints;
+    private Set<CommandInvocationNode> suspiciousPoints; // todo delete
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "suspiciousPoints" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)